src/test/java/com/akademia/educators_back/mapper/CommentMapperTest.java

```diff
@@ -0,0 +1,83 @@
+package com.akademia.educators_back.mapper;
+
+import com.akademia.educators_back.entity.CommentEntity;
+import com.akademia.educators_back.to.CommentTo;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+public class CommentMapperTest {
+
+    @Autowired
+    private CommentMapper commentMapper;
+
+    @Test
+    void shouldMapToEntityWithProperToValues() {
+        //given
+        CommentTo commentTest = CommentTo.builder()
+                .id(1L)
+                .answer("foo")
+                .build();
+       //when
+        CommentEntity commentEntity = commentMapper.toCommentEntity(commentTest);
+        //then
+        assertEquals(1L, commentEntity.getId());
+        assertEquals("foo", commentEntity.getAnswer());
+    }
+
+    @Test
+    public void testToDTO_NullInput() {
+        //when
+        CommentTo commentDTO = commentMapper.toCommentTO(null);
+        //then
+        assertNull(commentDTO);
+    }
+
+    @Test
+    void shouldMapToCommentWithProperCommentEntityValues() {
+        //given
+        CommentEntity commentEntity = CommentEntity.builder()
+                .id(1L)
+                .answer("foo")
+                .build();
+        //when
+        CommentTo commentTo = commentMapper.toCommentTO(commentEntity);
```

Comment:
a tutaj dziwnie..... w assercji nie sprawdzasz w ogóle tego commentTo, zakładam, że to kwestia Intellisense, popraw proszę